### PR TITLE
Custom health

### PR DESCRIPTION
**Pull request by fjoubert**
@theunsa 
Initial commit for the dynamic custom health view, not accessible by the user yet other than typing in the url /#/custom-health
Added roboto font
Some layout and styling fixes for the upcoming 0.9 material build (rc1 is out now)
Initial commit for accessing sensor graph

### CODE REVIEW

File: app/d3/d3-util-service.js [hunk 1]
```diff
@@ -78,15 +78,24 @@ angular.module('katGui.d3')
                     updateTooltipValues(d, tooltip);
                     var x = d3.event.layerX;
                     var y = d3.event.layerY;
-                    if (window.innerWidth - x < 280) {
-                        x = window.innerWidth - 280;
+                    //move the tooltip to 36,36 when we hover over the hide button
+                    if (d3.event.layerX - d.x < 24 && d3.event.layerY - d.y < 32) {
+                        //x = d3.x + 26;
```
Comment (theunsa) [inline app/d3/d3-util-service.js]: Just confirm that these lines should be commented out.

File: app/health/custom-health/custom-health.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+         ng-class="{'transparent-legend':transparentLegend && !showLegend}">
+        <md-button flex style="pointer-events: auto; width: 36px; height: 40px;" md-theme="{{themePrimaryButtons}}" ng-init="showLegend = false" ng-click="showLegend = !showLegend"
+                   ng-mouseenter="transparentLegend = false" ng-mouseleave="transparentLegend = true"
+                   aria-label="Show Legend" class="md-primary md-raised md-icon-button">
+            <md-icon md-theme="white" class="fa md-primary" ng-show="!showLegend" md-font-icon="fa-eye" title="Show Legend"></md-icon>
+            <md-icon md-theme="white" class="fa md-primary" ng-show="showLegend" md-font-icon="fa-eye-slash" title="Hide Legend"></md-icon>
+        </md-button>
+        <div class="md-whiteframe-z3" layout="column" ng-show="showLegend" style="background: white; padding: 8px;">
+            <div layout="row" layout-align="start center"><div class="color-patch unknown-child" title="Grey"></div><span>Unknown</span></div>
+            <div layout="row" layout-align="start center"><div class="color-patch nominal-child" title="Green"></div><span>Nominal</span></div>
```
Comment (martinslabber) [inline app/health/custom-health/custom-health.html]: Makes me fuzzy and happy inside that Python like to wrap code at 80 characters ........  ;)

File: app/d3/d3-util-service.js (hunk 1 above)
Comment (fjoubert) [inline app/d3/d3-util-service.js]: For the logic flow yes, but the if statement needs some rework, I've added a todo

File: app/health/custom-health/custom-health.html (hunk 2 above)
Comment (fjoubert) [inline app/health/custom-health/custom-health.html]: Would be unreadable if I had to wrap at 80 :P

**Pull request merged**